BREAKING_CHANGES.md

```diff
@@ -24,6 +24,16 @@
 
 - removed; see `@zendeskgarden/react-forms`
 
+## @zendeskgarden/react-forms
+
+- modified theming COMPONENT_IDs for alignment with `data-garden-id`
```

Comment:
Do we want to call these out under breaking changes? With us changing the structure and styling of the components these would probably break for people regardless.

Reply:
Probably fair to add a "blanket statement" to the effect that all referenced COMPONENT_IDs should be re-checked for naming accuracy.